generator/bin/3d-tiles-generator.js

```diff
@@ -195,6 +195,10 @@ var ulTileOptions = {
     relativeToCenter : relativeToCenter
 };
 
+// RequestVolumes
+var requestHeight = 50.0;
+var childRequestRegion = [longitude - longitudeExtent / 2.0, latitude - latitudeExtent / 2.0, longitude + longitudeExtent / 2.0, latitude + latitudeExtent / 2.0, 0.0, requestHeight];
+
```

Comment:
Fine to put these inside `createTilesetReplacementWithViewerRequestVolume` since they aren't used elsewhere.